app/src/main/dist/config/crate.yml

```diff
@@ -107,6 +107,17 @@ auth:
 #  trust:
 #    http_default_user: dustin
 
+#///////////////////////// External Authentication //////////////////////////////
+
+# Default values for JWT properties.
+# If a user is created without specifying JWT properties, default values are used.
+# CrateDB username is dictated by the 'username' property of a JWT token.
+#external_auth:
+#  jwt:
+#    iss: https://example.com
+#    aud: test_aud
```

Comment:
I think `aud` should probably be optional for the global configuration, similar to how it is for the per-user config, and it should default to the cluster ID in this case as well.

However, this raises the question: what should the behavior be if both the global `aud` value is set and there is also a user config? Should it use the global `aud` value or the cluster ID?

I would lean toward using the global `aud` value in this case.

> https://cratedb.com/docs/crate/reference/en/latest/admin/auth/methods.html
> If aud has not been defined on the CREATE USER statement, the cluster id is used and must match the token’s aud value

Reply:
>if both the global aud value is set and there is also a user config? Should it use the global aud value or the cluster ID?

> I would lean toward using the global aud value in this case.


I think user defined value must always take precedence over global, with the exception that "generated" should be weaker than concrete. Otherwise it's a bit messy that some configs are taken from user config and others from global.

ie 
1. user provided `aud` > global `aud`
2. user skipped `aud`:
   2.1 if global `aud` is defined, take it (instead of using auto-computed from the clusterID)
   2.2 global `aud` is not defined - use clusterId (pre 5.10 behaviour).
   
@mfussenegger, @henrikingo wdyt?

Reply:
Yes:  user provided -> global -> clusterId

Reply:
I didn't specifially highlight this case in the docs as I think @proddata's [suggestion](https://github.com/crate/crate/pull/16763#discussion_r1796007699) covers this.